src/edit/main.js

```diff
@@ -512,6 +512,15 @@ export class ProseMirror {
   markAllDirty() {
     this.dirtyNodes.set(this.doc, DIRTY_REDRAW)
   }
+
+  // :: (string) → string
+  // Return a translated string, if a translate function has been supplied,
+  // or the original string.
+  translate(string) {
```

Comment:
I'm not aware of a universal convention, but it might often be called
something like "t", "_t" or even "_". In this implementation, however, I
have kept the use of it in only a couple of central places where the
text is actually printed. This has it's pros and cons, I guess we will
see where we end up with that.

I would be fine with calling it "pm.t" if there is consensus on that.

On 2016-03-07 16:31, Marijn Haverbeke wrote:
>> @@ -512,6 +512,15 @@ export class ProseMirror {
>>    markAllDirty() {
>>      this.dirtyNodes.set(this.doc, DIRTY_REDRAW)
>>    }
>> +
>> +  // :: (string) → string
>> +  // Return a translated string, if a translate function has been supplied,
>> +  // or the original string.
>> +  translate(string) {
> 
> Is there any existing convention for a short name to use for this? I expect that in some types of code it'll show up a lot. `l10n` might be a bit too obscure.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ProseMirror/prosemirror/pull/248/files#r55219102
> 

-- 
http://yesbabyyes.se/
+46 709 89 03 85